src/tests/Interop/PInvoke/Int128/Int128TestFieldLayout.csproj

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <DebugType>embedded</DebugType>
```

Comment:
Is this necessary?

Reply:
The separate test? Yes, as the other test fails, and I'd like to not get rid of it. The DebugType flag, no, but it was just a copy/paste from the other project.

Reply:
The question was around the DebugType in particular.